src/OpenTelemetry.Collector.AspNetCore/RequestsCollector.cs

```diff
@@ -34,16 +34,16 @@ public class RequestsCollector : IDisposable
         /// Initializes a new instance of the <see cref="RequestsCollector"/> class.
         /// </summary>
         /// <param name="options">Configuration options for dependencies collector.</param>
-        /// <param name="tracer">Tracer to record traced with.</param>
+        /// <param name="tracerFactory">Factory for creating named Tracers.</param>
         /// <param name="sampler">Sampler to use to sample dependency calls.</param>
-        public RequestsCollector(RequestsCollectorOptions options, ITracer tracer, ISampler sampler)
+        public RequestsCollector(RequestsCollectorOptions options, ITracerFactory tracerFactory, ISampler sampler)
         {
             this.diagnosticSourceSubscriber = new DiagnosticSourceSubscriber(
                 new Dictionary<string, Func<ITracer, Func<HttpRequest, ISampler>, ListenerHandler>>()
                 {
                     { "Microsoft.AspNetCore", (t, s) => new HttpInListener(t, s) },
                 },
-                tracer,
+                tracerFactory.Create("Microsoft.AspNetCore"),
```

Comment:
shouldn't the name be "OpenTelemetry.Collector.AspNetCore" a.k.a. name of the instrumentation lib?